components/action-mgt/org.wso2.carbon.identity.action.management/src/main/java/org/wso2/carbon/identity/action/management/dao/impl/CacheBackedActionMgtDAO.java

```diff
@@ -125,4 +128,21 @@ public Action getActionByActionId(String actionId, Integer tenantId) throws Acti
 
         return actionManagementDAO.getActionByActionId(actionId, tenantId);
     }
+
+    @Override
+    public Action updateActionEndpointAuthProperties(String actionId, AuthType authentication, int tenantId)
+            throws ActionMgtException {
+
+        return actionManagementDAO.updateActionEndpointAuthProperties(actionId, authentication, tenantId);
```

Comment:
why doesn't the caches get cleared here

Reply:
Since in this scenario we are not updating any value in the IDN_ACTION and IDN_ACTION_ENDPOINT tables, we dont need to invalidate the cache. Only the values in the IDN_SECRET table are updating.